CONTRIBUTING.md

```diff
@@ -1,106 +1,186 @@
-## Contributing
+# Contributing
 
-| Component | Build Status |
-| --------- | ------------ |
-| Management Libraries | [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) |
-| Client Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master) |
+| Component            | Build Status                                                                                                                                                                              |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Management Libraries | [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net)                                                         |
+| Client Libraries     | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master) |
 
-### Prerequisites:
-  Install VS 2017 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
+# Prerequisites:
 
-### To build:
-=======
+Install VS 2019 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
 
-#### If you are building from VS, add a nuget feed source that points to < root >\tools\LocalNugetFeed directory
- 1. Open any solution, eg "Compute\Microsoft.Azure.Management.Compute.sln"
- 2. Build solution from VS
- 
-#### Full Build from command line
+**Client Libraries** are sdks used to interact with azure resources at application run time while **Management Libraries** are those used to manage (create/modify/delete) Azure resources.
+<br/><br/>
 
- 1. Open VS 2017 developer command prompt
- 2. Navigate to repository root directory
- 3. Invoke **msbuild** build.proj /t:Build
- will Build
- ##### *Build* without any scope will build all management SDK's.
+# Client Libraries
 
-#### Build single SDK
-`msbuild build.proj /t:CreateNugetPackage /p:scope=Compute`
+## TO BUILD:
+
+### Single Service from Command Line
+
+1. Open VS 2019 Command Prompt
+2. Navigate to service directory e.g. _"sdk\eventhub"_
+3. Invoke `dotnet build`
+4. or Build the **service.proj** in the repo root, passing the directory name of the specific service as a property. e.g. `dotnet build eng\service.proj /p:ServiceDirectory=eventhub`
```

Comment:
I'm making some changes to the root build.proj as part of https://github.com/Azure/azure-sdk-for-net/pull/6623 which should hopefully allow you to do this directly from the root with out the need to specify `eng\service.proj`

Reply:
Looks like build.proj builds both dataplane and management.